templates/robotics/index.html

```diff
@@ -35,24 +35,24 @@ <h2 class="p-heading--four">Tech visionaries choose Ubuntu to build ultra-secure
 
 <div class="p-strip--light">
   <div class="u-fixed-width">
-    {% with title="Proven, in the field - companies with robots on Ubuntu", json_feed_url="https://partners.ubuntu.com/partners.json?technology__name=Robotics", feed_item_limit=10 %}{% include "shared/_partner-logos.html" %}{% endwith %}
+    {% with title="Proven, in the field - companies using ROS on Ubuntu", json_feed_url="https://partners.ubuntu.com/partners.json?technology__name=Robotics", feed_item_limit=10 %}{% include "shared/_partner-logos.html" %}{% endwith %}
```

Comment:
Not something this PR introduced, but the line in the copy doc doesn't have the oddly placed comma:
```suggestion
    {% with title="Proven in the field - companies using ROS on Ubuntu", json_feed_url="https://partners.ubuntu.com/partners.json?technology__name=Robotics", feed_item_limit=10 %}{% include "shared/_partner-logos.html" %}{% endwith %}
```